src/terminologies/mdm-terminology.resource.ts

```diff
@@ -243,4 +244,21 @@ export class MdmTerminologyResource extends MdmModelDomainResource {
     const url = `${this.apiEndpoint}/admin/terminologies/${terminologyId}/undoSoftDelete`;
     return this.simplePut(url, {}, options);
   }
+
+  /**
+   * `HTTP PUT` - Copies a terminology to a new folder.
+   *
+   * @param terminologyId The unique identifier of the terminology to copy.
+   * @param CopyContainerPayload The payload to pass to the request when copying the terminology.
+   * @returns The result of the `PUT` request.
+   *
+   * `200 OK` - will return a {@link TerminologyDetailResponse} containing a {@link TerminologyDetail} object.
+   */
+  copy(
+    terminologyId: Uuid,
+    copyContainerPayload: CopyContainerPayload
```

Comment:
More consistency things:

- [ ] Move `target...` and `source...` properties into function parameters
- [ ] Make it clear which IDs are the source and which are the target
- [ ] `copyContainerPayload` is not the correct name for this, this implies we're copying a container (like a folder) whereas this is for a model. `CopyModelPayload` would make more sense